### ISSUE TITLE
 I couldn't seem to include the exclude this file

### ISSUE THREAD

**Issue by mgifford** [GovTechSG/purple-a11y#315]
When I added either:

  -x, --blacklistedPatternsFilename  

With

node --max-old-space-size=6000 --no-deprecation purple-a11y/cli.js -u https://www.whitehouse.gov -c 2 -s same-domain -p 50  -a none --blacklistedPatternsFilename ./pa-gTracker-exclude-medicare.csv -k "Random Example:random@example.com"  
  
It gave me the hel page.

**Comment by younglim** [comment]
Would you be able to elaborate :)

**Comment by mgifford** [comment]
I'm going to close this for now as I am having difficulties with the new script.  I will re-open if I can repeat this.
